cf-serverd/cf-serverd.c

```diff
@@ -61,7 +63,9 @@ int main(int argc, char *argv[])
         policy = LoadPolicy(ctx, config);
     }
 
+    GenericAgentPostLoadInit(ctx);
```

Comment:
Right, thanks! The code is indeed tricky, I'm not sure if anything is needed to be added there. Testing will tell...